config/elasticsearch.yml

```diff
@@ -21,9 +20,9 @@ http.enabled: ${HTTP_ENABLE}
 http.compression: true
 http.bind_host: ${HTTP_BIND_HOST}
 
-cloud.kubernetes.service: ${KUBERNETES_SERVICE}
-cloud.kubernetes.namespace: ${KUBERNETES_NAMESPACE}
-discovery.zen.hosts_provider: ${DISCOVERY_TYPE}
+s3.client.default.access_key: ${CLOUD_AWS_S3_ACCESS_KEY}
+s3.client.default.secret_key: ${CLOUD_AWS_S3_SECRET_KEY}
```

Comment:
Maybe move those to where the previous s3 config options were. Is this format now enforced / supported in 5.6?
What about region?

Reply:
Done.